### ISSUE TITLE
Deprecated function bug

### ISSUE THREAD

**Issue by IlyaSavich** [wapmorgan/PhpCodeFixer#23]
Hello, I'm using your package and facing with this case. 
```
> bin/phpcf --target=7.1 src
Max file size set to: 1.000 MiB
Scanning src ...
 PHP |             Type |                      File:Line | Issue
 5.3 | function         | ...er/VehicleController.php:70 | Function split() is deprecated. 
```

```
VehicleController.php:70

$options = $this->get('vs.options_splitter')->split($vehicle);
```

As I understand this fixer think about method as about function ?
Am I correct that it's bug ? Or I'm wrong ?

**Comment by wapmorgan** [comment]
Yes, it is bug. PhpCodeFixer (really, it does not fix your code) has simple scanning system, so cases like this are probable.

**Comment by wapmorgan** [comment]
Fixed in **dev-master**. Check it out.

**Comment by wapmorgan** [comment]
Released as **2.0.10**.
